travis/travis_install_nightly

```diff
@@ -108,21 +108,23 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 
```

Comment:
@moylop260 

In the line `110` shoul be add the follow lines

```bash
# Upgrade pip 
pip install --upgrade pip
```
That lines solved the follow mistake:

```bash
pip install -q --no-binary pycparser -r ${ODOO_PATH}/requirements.txt

Usage:   
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

no such option: --no-binary
```
Actually the version of pip is `pip 6.0.8` the upgrade version is `pip 9.0.1`